test/manipulation.test.js

```diff
@@ -688,6 +688,313 @@ describe('manipulation', function () {
     });
   });
 
+  if (!getSchema().connector.replaceOrCreate) {
+    describe.skip('replaceOrCreate - not implemented', function(done) { done; });
```

Comment:
Oh, I didn't realise we provide a non-atomic shim for connectors that don't provide an atomic version. However, this shim still depends on the connector to provide `replaceById` implementation, doesn't it?

In that case we should change this condition here - run the tests if the connector provides `replaceById`, skip them otherwise.

Here is my intention behind the original comment: I don't want to break CI builds of all connectors out there (including community ones) when this PR lands. At the same time, I want a notification for connector maintainers that their connector is not supporting all juggler features. `describe.skip` fits that well - `mocha -R spec` will print the test name in a different colour and don't fail the build.